### PR TITLE
fix(dcmjs): #231 Fix the normalize Alphabetic twice

### PR DESCRIPTION
**Pull request by wayfarer3130**
Fixed issue 231 where normalizing twice fails, and prevents side affects for modifying the array of length one, but doesn't fix any of the issues modifying longer non-sequence arrays, nor modifications to the properties of the child objects.

### CODE REVIEW

File: test/test_data.js [hunk 1]
```diff
@@ -115,6 +115,14 @@ function makeOverlayBitmap({ width, height }) {
 }
 
 const tests = {
+    test_array_items: () => {
+        const dicomJSON = JSON.stringify(require('./arrayItem.json'));
+        const datasets = JSON.parse(dicomJSON);
+        const natural0 = DicomMetaDictionary.naturalizeDataset(datasets[0]);
+        // Shouldn't throw an exception
+        const natural0b = DicomMetaDictionary.naturalizeDataset(datasets[0]);
```
Comment (pieper) [inline test/test_data.js:123]: Can you also test if they are the same?

**Review by pieper — CHANGES_REQUESTED** [review]
Can you add some more description in your commit message?  Why did the previous code lead to the issue, what changes did you make and why did they fix the problem?  It will help to have more context if this or a related issue comes up again.

File: test/test_data.js (hunk 1 above)
Comment (wayfarer3130) [reply test/test_data.js:123]: Yes, done.

**Comment by wayfarer3130** [comment]
> Can you add some more description in your commit message? Why did the previous code lead to the issue, what changes did you make and why did they fix the problem? It will help to have more context if this or a related issue comes up again.

Redid the commit message when I added the other fix and updated to master.

**Review by pieper — APPROVED** [review]
Thanks for the updates 👍

**Pull request merged**